nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -69,6 +70,57 @@ def inject_dataloader_value_from_model_config(model_cfg: dict, dataloader_cfg: D
             dataloader_cfg[key] = model_cfg[key]
 
 
+def get_concat_char_dataset(
+    config: dict, global_rank: int, world_size: int, augmentor: Optional['AudioAugmentor'] = None
+) -> ConcatDataset:
+    """
+    Instantiates an instance of ConcatDataset containing one or more intances of
+    Character Encoding based AudioToCharDataset.
+
+    Args:
+        config: Config of the AudioToCharDataset.
+        global_rank: Global rank of this device.
+        world_size: Global world size in the training method.
+        augmentor: Optional AudioAugmentor object for augmentations on audio data.
+
+    Returns:
+        An instance of ConcatDataset containing one or more instances of AudioToCharDataset.
+    """
+    if 'labels' not in config:
+        logging.warning(f"dataset does not have explicitly defined labels")
+
+    manifest_filepaths = config['manifest_filepath']
+    datasets = []
+    for manifest_filepath in manifest_filepaths:
+        dataset = audio_to_text.AudioToCharDataset(
```

Comment:
Hmm instead of duplication how about passing these configs to the underlying methods defined for chat / subword and then using the result for concat

Reply:
++